script_runner/tasks.py

```diff
@@ -69,15 +74,19 @@ def run(script_path, process=None, **kwargs):
     script_path = download_resource(ctx.download_resource, script_path)
     os.chmod(script_path, 0755)
     script_func = get_run_script_func(script_path, process)
-    return process_execution(script_func, script_path, ctx, process)
+    script_result = process_execution(script_func, script_path, ctx, process)
+    os.remove(script_path)
```

Comment:
Nope. If something breaks during execution, I certainly don't want to delete the script — so I can troubleshoot things.